favorites.py

```diff
@@ -218,6 +217,7 @@ def load_favorite_files(cls, obj, force=False, clean=False, win_id=None):
             else:
                 errors = True
 
+        cls.check_plugin_version(obj.file_name)

```

Comment:
Forgot here to check for errors:

        # Update json if necessary
        if not errors:
            cls.check_plugin_version(obj.file_name)